quiz-microsevice/cli/index.js

```diff
@@ -32,6 +37,22 @@ cli
           await retrieveAllClients();
           break;
 
+// I've changed the vars name (ID, Id) because these vars is locally, so it's make conflict.
```

Comment:
To ensure that each `case` has its own scope for `const`, it's best to wrap the code for each `case` in curly braces `{}`.

```
switch (value) {
  case 1: {
    const x = 10;  // Scoped to this block
    console.log(x);
    break;
  }
  case 2: {
    const x = 20;  // Scoped to this block
    console.log(x);
    break;
  }
}
```